components/packages/comments/src/index.tsx

```diff
@@ -35,7 +45,9 @@ const ReplyIcon = () => {
   )
 }
 
-const Comment = forwardRef<HTMLDivElement, CommentProps>(
+export type CommentForwardedRef = HTMLDivElement & { highlight: () => void }
```

Comment:
```suggestion
export type CommentForwardedRef = HTMLDivElement & { highlight: VoidFunction }
```
`+` в `types.ts` вынести бы

Reply:
fixed